packages/paste-nextjs-template/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@twilio-paste/nextjs-template",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@twilio-paste/core": "^4.2.4",
+    "@twilio-paste/icons": "^3.10.5",
+    "next": "^10.0.4",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
```

Comment:
Should we add web-vitals to this template too?

Reply:
Might give it a try

Reply:
Whoa, it's basically built in already https://nextjs.org/docs/advanced-features/measuring-performance